internal/pkg/encoding/json/schema/schema.go

```diff
@@ -126,31 +129,32 @@ func processErrors(errs []*jsonschema.ValidationError) error {
 	schemaErrs := errors.NewMultiError()
 	docErrs := errors.NewMultiError()
 	for _, e := range errs {
-		// Process nested errors
-		if len(e.Causes) > 0 {
+		isSchemaErr := !strings.HasPrefix(e.AbsoluteKeywordLocation, "file://"+pseudoSchemaFile)
```

Comment:
Prvy commit, vylepsena detekcia toho ci jedna o chybu v definicii JSON schemy, alebo v samotnom dokumente.

Zial, pouzita libka nevie dostatocne zvalidovat samotnu schemu, ale da sa dobre detekovat, kde chyba nastala.

Dotera sme to robili tak (vid `HasSuffix` nizsie v starom kode), ze ak cesta ku chybe obsahuje `https://json-schema.org/`, tak sa jedna o chybu pri validacii samotnej schemy. Lenze, ak si user definuje vlastny `draft`, tak tam moze byt aj `http` -> tj. je to nespolahlive.

Preto som logiku otocil. Ak chyba nevznikla na zaklade nasej schemy, tak sa jedna o chybu v samotnej scheme.